src/ChannelServer/Network/Handlers/Parties.cs

```diff
@@ -0,0 +1,308 @@
+﻿// Copyright (c) Aura development team - Licensed under GNU GPL
+// For more information, see license file in the main folder
+
+using Aura.Channel.Network.Sending;
+using Aura.Channel.Util;
+using Aura.Channel.World.Entities;
+using Aura.Data;
+using Aura.Data.Database;
+using Aura.Mabi.Const;
+using Aura.Mabi.Network;
+using Aura.Shared.Network;
+using System;
+
+namespace Aura.Channel.Network.Handlers
+{
+    public partial class ChannelServerHandlers : PacketHandlerManager<ChannelClient>
```

Comment:
Was it not you who wanted me to use spaces instead of ta- Ohhh, that was the other way around, and it was in VS2012, so that setting wouldn't have come across to 2015, wooops.

In any case, space people made a pretty good argument.